.github/actions/godot-build/action.yml

```diff
@@ -0,0 +1,39 @@
+name: 'Build Godot'
```

Comment:
In the various actions, the `name` and `description` tags have inconsistent styling when it comes to:
- Ending sentences with a dot
- Using single quotes, or no quotes

If the single quotes are optional, maybe we should standardize on:
- No dots
- No quotes

WDYT?

Reply:
> No dots

Or maybe no dots in `name`s, and dots in `description`s.

Reply:
Done